playbooks/roles/shadowsocks/vars/main.yml

```diff
@@ -32,6 +32,11 @@ shadowsocks_location: "/etc/shadowsocks-libev"
 shadowsocks_password_file: "{{ shadowsocks_location }}/shadowsocks-password.txt"
 shadowsocks_password_file_localpath: "../{{ streisand_local_directory }}/shadowsocks-password.txt"
 
+#simple-obfs vars
+simple_obfs_version: "0.0.2"
```

Comment:
There is actually a newer "0.0.3" release. Can you update this PR to use that release instead?

I opened [a PR with upstream](https://github.com/shadowsocks/simple-obfs/pull/75) to fix the README, which still claims that 0.0.2 is the latest release.